.gitignore

```diff
@@ -18,8 +18,8 @@ media/
 /meinberlin/static/datepicker.*
 /meinberlin/static/embed.js
 /meinberlin/static/fonts/
-/meinberlin/static/images/layers-2x.png
-/meinberlin/static/images/layers.png
-/meinberlin/static/images/spritesheet-2x.png
-/meinberlin/static/images/spritesheet.*
+/images/layers-2x.png
+/images/layers.png
+/images/spritesheet-2x.png
+/images/spritesheet.*
```

Comment:
ah sorry, i have been on an unmerged branch and wondered why git was showing the files